shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-opengauss/src/main/java/org/apache/shardingsphere/sql/parser/opengauss/visitor/statement/impl/OpenGaussDDLStatementSQLVisitor.java

```diff
@@ -747,7 +750,35 @@ public ASTNode visitDeclare(final DeclareContext ctx) {
     }
     
     @Override
+    @SuppressWarnings("unchecked")
     public ASTNode visitComment(final CommentContext ctx) {
-        return new OpenGaussCommentStatement();
+        OpenGaussCommentStatement result = new OpenGaussCommentStatement();
+        if (null != ctx.commentClauses().objectTypeAnyName() && null != ctx.commentClauses().objectTypeAnyName().TABLE()) {
+            LinkedList<NameSegment> nameSegments = (LinkedList<NameSegment>) ((CollectionValue<NameSegment>) visit(ctx.commentClauses().anyName())).getValue();
```

Comment:
Please replace LinkedList with Collection.

Reply:
Thank you. Done.